tensorbay/cli/branch.py

```diff
@@ -5,22 +5,20 @@
 
 """Implementation of gas branch."""
 
-import sys
 from typing import Dict
 
 import click
 
 from ..client.gas import DatasetClientType
 from .tbrn import TBRN, TBRNType
-from .utility import get_dataset_client, get_gas
+from .utility import errors_handler, get_dataset_client, get_gas
```

Comment:
the commit message should be updated,
current commit message is not easy to understand. The word `properly` is obscure, I do not know how do you handle the error from the word `properly`.